docs.md

```diff
@@ -23,72 +23,82 @@ permalink: /docs/
 
 ## New to Flutter?
 
-* [Write Your First Flutter App](/get-started/codelab/)  
-  A walkthrough on creating a Flutter app that generates names for startup companies.
+Start here.
 
-* [Flutter for Android Developers](/flutter-for-android/)  
-  Tips for Android developers.
+* [Write Your First Flutter App](/get-started/codelab/)<br>
```

Comment:
FWIW, in Markdown, the `<br>` at the end can be replaced by just adding two spaces at the end. I like the explicitness of `<br>`, though, so feel free to keep this.

Reply:
I did not know that!!!!